db/migrate/20161107112025_remove_available_currencies_column_from_communities.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveAvailableCurrenciesColumnFromCommunities < ActiveRecord::Migration
+  def up
+    remove_column :communities, :available_currencies
+  end
```

Comment:
I intentionally left the rollback migration out since the data is lost anyways. Does it still make sense to add the column back if it loses the original data?